src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -354,6 +355,24 @@ public function iProceededWithShippingMethod(ShippingMethodInterface $shippingMe
         $this->ordersClient->executeCustomRequest($request);
     }
 
+    /**
+     * @When I try to select :paymentMethodCode payment method
+     */
+    public function iTryToSelectPaymentMethod(string $paymentMethodCode): void
```

Comment:
it is a duplication of method `iChoosePaymentMethod` can you use it instead of making a new one?

Reply:
It is not in 100%, the call is same (endpoint) but content is differently created see: `        $request->setContent(['paymentMethod' => $this->iriConverter->getItemIriFromResourceClass(PaymentMethod::class, ['code' => $paymentMethodCode])]);
`